unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/MainChatNotificationsComponentView.cs

```diff
@@ -0,0 +1,221 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DCL;
+using DCL.Interface;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class MainChatNotificationsComponentView : BaseComponentView
+{
+    [SerializeField] RectTransform chatEntriesContainer;
+    [SerializeField] GameObject chatNotification;
+    [SerializeField] ScrollRect scrollRectangle;
+    [SerializeField] Button notificationButton;
+    [SerializeField] public TMP_Text notificationMessage;
+
+    public ChatNotificationController controller;
```

Comment:
try to decouple the controller.. you can check it out how are we doing it on `feat/social-lazy-loading` implementation or `feat/channels`